Sender/BMS_Streaming/BatteryParameters.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BMS_Streaming
+{
+    public class BatteryParameters
```

Comment:
This class has only setters and getters. Can we encapsulate more logic in here? Like the parsing or output. Then adding new parameters gets localized to this class.